README.md

```diff
@@ -1,7 +1,135 @@
+to the Organization Settings screen. Copy your Organization ID and save it with
+your credentials.
+
+That's everything for Atlas. Now you're ready to move on to setting up Google Cloud!
+
+### Google Cloud
+
+If you don't already have a Google Cloud account, [sign up here](https://accounts.google.com/SignUp).
+We recommend using [Cloud Shell](https://cloud.google.com/shell) but if you prefer
+to work locally you can do that too. If you'd like to use Cloud Shell, just click
+this convenient button:
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://shell.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/terraform-mean-cloudrun-mongodb)
+
+This will open a Cloud Shell window and clone a copy of this repository. Cloud
+Shell has your credentials built in, and Terraform is already installed, so
+you're ready to move on to running the demo. You can also [open Cloud Shell the normal way](https://cloud.google.com/shell/docs/using-cloud-shell)
+but you'll need to clone this repo yourself.
+
+If you want to work locally, you'll need to [install the `gcloud` tool](https://cloud.google.com/sdk/docs/install).
+Confidential to Mac users: `gcloud` is also available [via homebrew](https://formulae.brew.sh/cask/google-cloud-sdk).
+Once the install is done, run `gcloud init` to set up your environment. More
+information can be found [in the docs](https://cloud.google.com/sdk/docs/initializing).
+
+### Installing Terraform
+
+If you're using Cloud Shell, you've already got Terraform installed so feel free
+to move on to the next section. Otherwise, [download Terraform](https://www.terraform.io/downloads)
+and install it. A detailed walkthrough can be found [in this tutorial](https://learn.hashicorp.com/tutorials/terraform/install-cli).
+
+### Choosing a region
+
+Next, select a region for your infrastructure. For this demo it's okay to choose
+somewhere close to you, but for production use you may want to choose a different
+region. If you need some advice, check out [Best Practices for Compute Engine regions selection](https://cloud.google.com/solutions/best-practices-compute-engine-region-selection).
+
+For a list of available regions, refer Atlas's [Google Cloud provider documentation](https://www.mongodb.com/docs/atlas/reference/google-gcp/).
+Choose a region that's close to you and that supports the `M0` cluster tier. Make
+a note of both the Google Cloud region name and the Atlas region name, as you'll
+need both in the next step.
+
+### Configuring the demo
+
+If you haven't already, clone this repo. Run `terraform init` to make sure
+Terraform is working correctly and download the provider plugins.
+
+Then, create a file in the root of the repository called `terraform.tfvars` with
+the following contents, replacing placeholders as necessary:
+
+    atlas_pub_key        = "<your Atlas public key>"
+    atlas_priv_key       = "<your Atlas private key>"
```

Comment:
Can `terraform.tfvars` be another directory and the path specified when invoking Terraform? With a private key in the contents, I imagine we want this in the user's home directory.

Reply:
Yeah, you can pass in an arbitrary tfvars file using a flag that's called something like `-var-file=`, but there are definitely some tradeoffs and I'm not sure it's worth it in this situation. I think given that it's a relatively simple demo, most people are likely to be cloning into their homedirs anyway, running it, going "oooooo neat", and then shutting it back down.

Reply:
Agree, most clones will already be under a home directory. I already had one commit mishap today, though, so I guess I'm just being extra wary of keys in a repo right now.

Reply:
Yeah, I hear you. In this case [we ignore those files](https://github.com/GoogleCloudPlatform/terraform-mean-cloudrun-mongodb/pull/8/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R6-R7) and others that would be automatically loaded so it takes a few extra steps to get into a situation where you might accidentally commit creds.

I think given the goals of being a low-friction demo it makes the most sense to aim for simple invocations and taking advantage of quality of life features. If someone wants to repurpose this demo code for production purposes they can make decisions about how to handle input in a way that makes sense for their org and policies.